internal/pkg/store/ssm/ssm_test.go

```diff
@@ -28,6 +28,8 @@ func TestStore_ListProjects(t *testing.T) {
 	cowProjectString, err := marshal(cowProject)
 	require.NoError(t, err, "Marshal project should not fail")
 
+	lastPageInPaginatedResp := false
```

Comment:
Should we set this back to `false` inside each `t.Run`? so that we can have more than one testcase that can have paginated responses